app/javascript/application.js

```diff
@@ -1,3 +1,3 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
-import "@hotwired/turbo-rails"
-import "controllers"
+// import "@hotwired/turbo-rails"
+// import "controllers"
```

Comment:
May I know the reason to disable these two modules? I don't really understand why they were here though.